include/quill/backend/BackendWorker.h

```diff
@@ -818,14 +817,71 @@ class BackendWorker
       detail::log_level_to_string(transit_event.log_level(), _options.log_level_short_codes.data(),
                                   _options.log_level_short_codes.size());
 
+    if (transit_event.logger_base->pattern_formatter->get_add_metadata_to_multi_line_logs() &&
+        (!transit_event.named_args || transit_event.named_args->empty()))
+    {
+      // This is only supported when named_args are not used
+      _process_multi_line_message(transit_event, log_level_description, log_level_short_code);
+    }
+    else
+    {
+      // if the log_message ends with \n we should exclude it
+      size_t const log_message_size =
+        transit_event.formatted_msg.data()[transit_event.formatted_msg.size() - 1] == '\n'
+        ? transit_event.formatted_msg.size() - 2
```

Comment:
@odygrd maybe I'm missing something by why did you subtract 2 here instead of 1?
It deletes the character before newline also

Reply:
I don’t remember to be honest, but I will double check it

Reply:
Thanks a lot for catching that! Fixing in https://github.com/odygrd/quill/pull/726